sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/tests/Tests/PrivateEndpointConnectionCollectionTests.cs

```diff
@@ -22,13 +22,11 @@ public class PrivateEndpointConnectionCollectionTests : AppConfigurationClientBa
         private ResourceGroupResource ResGroup { get; set; }
         private AppConfigurationStoreResource ConfigStore { get; set; }
         private Network.PrivateEndpointResource PrivateEndpointResource { get; set; }
-
         public PrivateEndpointConnectionCollectionTests(bool isAsync)
-            : base(isAsync)
+            : base(isAsync)//, RecordedTestMode.Record)
         {
         }
 
-        [SetUp]
         public async Task TestSetUp()
```

Comment:
Since this method is no longer a "set up", could we rename this method to make it more specific?

Reply:
fixed